raven/contrib/__init__.py

```diff
@@ -2,7 +2,7 @@
 raven.contrib
 ~~~~~~~~~~~~~
 
-:copyright: (c) 2010-2013 by the Sentry Team, see AUTHORS for more details.
```

Comment:
All these are `.py` files, I think it would be great to leverage python `date` module and generate valid date.

Reply:
honestly id rather just rm -rf all of these headers